apis/kubernetes/v1alpha1/containerregistry_types.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DOContainerRegistryParameters define the desired state of a DigitalOcean Container Registry.
+// Most fields map directly to a Containe rRegistry:
+// https://docs.digitalocean.com/reference/api/api-reference/#tag/Container-Registry
+type DOContainerRegistryParameters struct {
+	// The slug of the subscription tier to sign up for.
+	// Valid values can be retrieved using the options endpoint.
+	// +immutable
```

Comment:
Does this need to be immutable? We can update the subscription slug via the API so I'd think that we'd want it to be changeable as well.

Reply:
That's true, it does not have to be immutable.